src/main/java/frc/robot/subsystems/Turret.java

```diff
@@ -31,10 +31,13 @@ public Turret(){
    * Moves the Turret
    * @param speed the speed at which to set the turret motor to
    */
-
   public void set(double speed){
-    m_turretMotor.set(ControlMode.PercentOutput, speed);
-  }
+    if(getAngle() > Constants.TURRET_ROTATION_MIN && getAngle() < Constants.TURRET_ROTATION_MAX) {
+      m_turretMotor.set(ControlMode.PercentOutput, speed);
+    } else {
+      stop();
+    }
+  } 
```

Comment:
Instead of stopping whenever the turret is outside the acceptable range, why don't you instead only allow inputs that move it back into the acceptable range.

Reply:
Fixed in commit 80861ac